internal/statesync/chunks.go

```diff
@@ -355,3 +355,16 @@ func (q *chunkQueue) WaitFor(index uint32) <-chan uint32 {
 
 	return ch
 }
+
+func (q *chunkQueue) chunkReturnedSize() uint32 {
```

Comment:
I was confused by this name, expecting the result to be a size (e.g., bytes) rather than a count. Perhaps `numChunksReturned` or `chunkReturnedCount` would be clearer here?

On a related note: I see this reflects some existing usage in the package, but it's a little odd to explicitly choose integer size for counts of things that fit in memory. That's the use case `int` (undecorated) is designed for, as it will be chosen by the compiler to be big enough to reach the address space.

Having an unsigned index is also slightly unusual (though not wrong): It means any arithmetic someone might do on the value has to take care not to overflow downward, if sizes are ever subtracted. That might not be an issue here, but it stuck out.

Reply:
return to `numChunksReturned` and return `int` instead of `uint32`